lib/v2/hunau/maintainer.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    '/jwc/:type?/:page?': ['lcandy2'],
+    '/gfxy/:type?/:page?': ['lcandy2'],
+    '/ied/:type?/:category?/:page?': ['lcandy2'],
```

Comment:
```suggestion
    '/gfxy/:type?/:page?': ['lcandy2'],
    '/ied/:type?/:category?/:page?': ['lcandy2'],
    '/jwc/:type?/:page?': ['lcandy2'],
```
Sort entries in alphabetical order.
docs: https://docs.rsshub.app/joinus/script-standard.html